src/Sarif.Multitool.Library/Rules/RuleId.cs

```diff
@@ -49,6 +49,15 @@ public static class RuleId
         public const string ProvideCheckoutPath = "GH1006";
         public const string ProvideFullyFormattedMessageStrings = "GH1007";
 
+        // Rules for ADO Advanced Security.
+        public const string ADOMaximumRunsCount = "ADO1001";
```

Comment:
Looks like there is an extra space before "Security".

Reply:
I don't see it. Are you referring to "ADO Advanced Security"?

Reply:
Yes

Reply:
Just a single space...